testing/fasttext_test.py

```diff
@@ -169,4 +168,63 @@ def test_fasttext_boolean(self):
                 ]
             ])
 
+        def test_fasttext_explain(self):
+
+            mldb.log("explain")
+
+            cls_config = {
+                "my_fasttext": {
+                    "type": "fasttext",
+                    "verbosity" : 0,
+                    "dims" : 4,
+                    "epoch" : 5,
+                }
+            }
+
+            mldb.put("/v1/procedures/trainer", {
+                "type": "classifier.train",
+                "params": {
+                    "trainingData": "SELECT {tokens.*} as features, Theme as label FROM bag_of_words",
+                    "modelFileUrl": "file://tmp/src_fasttext_explain.cls",
```

Comment:
To avoid conflicts and cleanup results, it's better to use tempfiles.
```
import os
import tempfile
tmp_file = tempfile.NamedTemporaryFile(prefix=os.getcwd() + '/build/x86_64/tmp/')
```
And then here you can have
```
"modelFileUrl": "file:///" + tmp_file.name
```